### PR TITLE
minor: fix bugs about filename and boundary

### PR DESCRIPTION
**Pull request by HuGanghui**
#54  minor: fix bugs about filename and boundary

ALL Test Pass

### CODE REVIEW

File: src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ private boolean isLineMatching(AuditEvent event) {
                 final int startLine = aLineRangeList.get(0);
                 final int endLine = aLineRangeList.get(1);
                 final int currentLine = event.getLine();
-                result = currentLine > startLine && currentLine <= endLine;
+                result = currentLine >= startLine && currentLine <= endLine;
```
Comment (romani) [inline src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java:81]: please provide patch, input, config on where you caught it. Bug always result in extension of test code base.

Comment (rdiachenko) [reply src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java:81]: @HuGanghui my initial comment - https://github.com/checkstyle/patch-filters/pull/28#discussion_r444174777

> result = currentLine >= startLine && currentLine <= endLine;

Here's your second attempt to apply what I suggested - https://github.com/checkstyle/patch-filters/pull/39/files

> result = currentLine > startLine && currentLine <= endLine;

Please give attention to each single symbol next time you see proposed changes. If you don't agree or don't understand then ask.

Comment (HuGanghui) [reply src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java:81]: @rdiachenko I got it, I will be careful.

Comment (HuGanghui) [reply src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java:81]: @romani boundary test is here https://github.com/checkstyle/patch-filters/blob/27fb2668fc595a8db86896adecd7a572814e0397/src/test/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterTest.java#L83-L92 
https://github.com/checkstyle/patch-filters/blob/27fb2668fc595a8db86896adecd7a572814e0397/src/test/resources/com/github/checkstyle/patchfilter/BoundaryTestPatchOne.txt#L1-L5

Comment (HuGanghui) [reply src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java:81]: sorry, it's my oversight, sometimes I forget to execute the unit tests, and in the future I will add an ALL Test Pass to each PR description to ensure that ALL tests Pass.

**Comment by rdiachenko** [comment]
@HuGanghui 
> please create separate exact issue. and reference it in PR description.
it is not good to reference already closed issues.

Please do

Also, please rebase on latest master. We added build test, so your PR will fail if `mvn install` fails and you'll need to fix it.

File: src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java (hunk 1 above)
Comment (romani) [reply src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java:81]: > boundary test is here, this example is only one line changed, so the startLine and endLine is same, when we change isLineMatching to

if you change main code you change test code also, please provide test/input/.... to cover this case.

Comment (HuGanghui) [reply src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java:81]: ok

**Comment by HuGanghui** [comment]
@rdiachenko done

**Pull request merged**